contributing/contributing-code.md

```diff
@@ -175,3 +175,36 @@ These guidelines are specific to the amphtml repository. Other ampproject repos
   - When creating a new directory (such as when creating a new AMP extension) the author of the pull request should designate themselves as an Owner of that directory.
   - Owners of an area may approve other Owners at or below their area of expertise following the normal PR process. To request becoming an Owner create a PR adding yourself to the appropriate OWNERS file and assign/cc a current Owner for that directory.
 - The list of Owners for a directory can be found in the [OWNERS](https://github.com/ampproject/amphtml/search?o=asc&q=filename%3A"OWNERS"&s=indexed) file in the directory or a parent directory.
+
+## Cherry-picks
+
+We have a well-defined process for handling requests for changes to the **experimental**/**beta**, **stable**, or **lts** release builds. These changes are known as "cherry-picks".
+
+**The bar for getting a cherry-pick into a live release is very high** because our goal is to produce high quality launches on a predictable schedule.
+
+**Keep in mind that performing a cherry-pick requires a significant amount of work from you and the on-duty engineer** and they can take a long time to process.
+
+- In general only fixes for [P0 issues](https://github.com/ampproject/amphtml/blob/master/contributing/issue-priorities.md) (causing "an outage or a critical production issue") may be cherry-picked. P0 issues are those that:
+  - cause privacy or security issues
+  - cause user data loss
+  - break existing AMP web pages in a significant way
+  - or would otherwise cause a significant harm to AMP's reputation if left unresolved
+- Regressions found in the **experimental**/**beta** releases that are not P0 _may_ be approved if they can be resolved with a rollback. Fixes other than rollbacks--no matter how simple they may seem--will not be approved because these have the potential to cause cascading problems and delay the release promotion of **beta** to **stable** for everyone.
+
+### Process for requesting a cherry-pick
+
+Use the following process to request a cherry-pick if you have a change that you believe meets the cherry-pick criteria.
+
+If you run into any issues or have any questions when requesting a cherry-pick, please use the [AMP Slack #release channel](https://amphtml.slack.com/messages/C4NVAR0H3/) ([sign up for Slack](https://bit.ly/amp-slack-signup)).
+
+- Ensure there is a GitHub issue filed for the problem that needs to be resolved _before_ filing the cherry-pick request issue.
```

Comment:
Good opportunity to link out to the GH issues.

Reply:
Done.

Reply:
Done (in follow-up)